### ISSUE TITLE
[Question] What about `playwright-test`?

### ISSUE THREAD

**Issue by fschoenfeldt** [playwright-community/jest-playwright#408]
I just came across https://github.com/microsoft/playwright-test

What are the differences between `jest-playwright` and `playwright-test`? The syntax seem very similar.

**Comment by mxschmitt** [comment]
Hey,
once we built jest-playwright there was no playwright-test and folio for running tests. We created it to provide an easy solution to write tests in a common environment (Jest). The way of writing tests is a bit different. Playwright-test uses test fixtures which are e.g. used in Pytest too (named deconstructed values are passed into the test). Jest-Playwright uses globals to access the page and context. Both share their way of making assertions with expect.

We would recommend to use [playwright-test](https://github.com/microsoft/playwright-test) now if you don't have a test ecosystem in place since its created to provide first class support for Playwright. If you are already using Jest, it should also be fine to use jest-playwright. The core concepts of custom launch/context options, skip browsers, new context for each page they both share.
